WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JPScrollViewDelegate.swift

```diff
@@ -1,10 +1,11 @@
 import Combine
 import CoreGraphics
+import UIKit
 
 /// Conform to this protocol to send scrollview translations to a `JetpackBannerView` instance
 protocol JPScrollViewDelegate: UIScrollViewDelegate {
 
-    var scrollViewTranslationPublisher: PassthroughSubject<CGFloat, Never> { get }
+    var scrollViewTranslationPublisher: PassthroughSubject<Bool, Never> { get }
```

Comment:
We should probably update these names to better fit their purpose:
- `scrollViewTranslationPublisher`
- `addTranslationObserver`